### PR TITLE
added sidebar componenet

### PR DESCRIPTION
**Pull request by narharikale**
- added sidebar component 

screenshot
screenshot

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
Good color variables naming,
Looks Good.

File: src/frontend/components/sidebar/sidebar.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    align-items: flex-start;
+    
+}
+.sidebar-link{
+    display: flex;
+    gap: 2rem;
+    min-width: 12.5rem;
+    padding: 1.125rem 1.313rem;
+    align-items: center;
+    border-radius: 0rem 0.938rem 0.938rem 0rem;
```
Comment (jeetsdev) [inline src/frontend/components/sidebar/sidebar.css:15]: Why you are using such random numbers here, Is there any specific reason ...?

File: src/frontend/screens/home/home.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
+import { Sidebar } from "../../components/sidebar/sidebar";
+import './home.css'
+
+
+
+
+
+
```
Comment (jeetsdev) [inline src/frontend/screens/home/home.jsx]: Please remove these extra lines...

**Review by jeetsdev — APPROVED** [review]
Everything looks good, Added some suggestions though.

File: src/frontend/components/sidebar/sidebar.css (hunk 1 above)
Comment (narharikale) [reply src/frontend/components/sidebar/sidebar.css:15]: no i just converted these from px to rem

File: src/frontend/screens/home/home.jsx (hunk 2 above)
Comment (narharikale) [reply src/frontend/screens/home/home.jsx]: sure

Comment (vaishnavipaliya11) [inline src/frontend/screens/home/home.jsx]: remove these empty spaces

**Review by vaishnavipaliya11 — APPROVED** [review]

File: src/frontend/components/sidebar/sidebar.css (hunk 1 above)
Comment (vaishnavipaliya11) [inline src/frontend/components/sidebar/sidebar.css:15]: as a suggestion you should use another unit if it is less than 1rem

**Review by vaishnavipaliya11 — APPROVED** [review]

**Comment by vaishnavipaliya11** [comment]
Code looks clean. Naming conventions are quite good.

**Comment by vaishnavipaliya11** [comment]
There are no issues you can go ahead

File: src/frontend/components/sidebar/sidebar.css (hunk 1 above)
Comment (narharikale) [reply src/frontend/components/sidebar/sidebar.css:15]: sure i will implement in future use case

File: src/frontend/screens/home/home.jsx (hunk 2 above)
Comment (narharikale) [reply src/frontend/screens/home/home.jsx]: yes fixed !

**Pull request merged**